### PR TITLE
Added Airdrop to dali runtime

### PR DESCRIPTION
**Pull request by PoisonPhang**
## Issue
Blocking #2g9jvc9

## Description
Added Airdrop to the dali runtime. Needs benchmarks

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2g9jvc9 Airdrop pallet](https://app.clickup.com/t/2g9jvc9)

File: runtime/dali/src/weights/airdrop.rs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::Weight};
+use sp_std::marker::PhantomData;
+
+/// Weight functions for `bonded_finance`.
```
Comment (benluelo) [inline runtime/dali/src/weights/airdrop.rs:8]:
```suggestion
/// Weight functions for `airdrop`.
```

Comment (PoisonPhang) [reply runtime/dali/src/weights/airdrop.rs]: 3b831088f9ff792e8a53a4daf1e8c958e37a3982

File: runtime/dali/src/weights/airdrop.rs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		0
+	}
+
+	fn disable_airdrop() -> Weight {
+		0
+	}
+
+	fn claim(_x: u32) -> Weight {
+		0
+	}
```
Comment (KaiserKarel) [inline runtime/dali/src/weights/airdrop.rs:33]: Seems like we still need to get this benchmarked?

**Review by KaiserKarel — CHANGES_REQUESTED** [review]
Needs benchmarking?

**Comment by PoisonPhang** [comment]
> Needs benchmarking?

It will. Going to address fixes before requesting benchmarks again

**Comment by PoisonPhang** [comment]
Closing for now while Airdrop is in the backlog

**Pull request closed without merging**